src/tracks/gosling-track/gosling-track.ts

```diff
@@ -830,7 +824,7 @@ export class GoslingTrackClass extends TiledPixiTrack<Tile, GoslingTrackOptions>
      */
     override removeOldTiles() {
         super.removeOldTiles(); // some items are added to this.fetching
-        if (!isTabularDataFetcher(this.dataFetcher)) {
+        if (isTabularDataFetcher(this.dataFetcher)) {
```

Comment:
Another bug fix that comes from `v2`. This change enables displaying json-based tracks.